cadc-test-vos/src/main/java/ca/nrc/cadc/conformance/vos/VOSTransferTest.java

```diff
@@ -256,13 +237,15 @@ protected TransferResult doAsyncTransfer(Transfer transfer)
         assertNotNull("Location header not set", location);
 
         // Follow the redirect.
-        while (303 == response.getResponseCode())
-        {
-            location = response.getHeaderField("Location");
-            assertNotNull("Location header not set", location);
-            log.debug("New location: " + location);
-            response = get(location);
-        }
+        // TOOD: remove this and other commmented code when the functions are proven to work
```

Comment:
Not sure if you meant to leave this TODO in there...

Reply:
removed this